code/__DEFINES/genetics.dm

```diff
@@ -96,6 +96,7 @@
 #define NUTRITION_LEVEL_FED 350
 #define NUTRITION_LEVEL_HUNGRY 250
 #define NUTRITION_LEVEL_STARVING 150
+#define NUTRITION_LEVEL_CURSED -1000
```

Comment:
Nutrition is never supposed to go below 0, so this should be 0 instead of -1000.